clients/admin-ui/src/features/common/nav/v2/MainSideNav.tsx

```diff
@@ -120,34 +131,103 @@ const NavGroupMenu = ({
 export const UnconnectedMainSideNav = ({
   groups,
   active,
+  handleLogout,
+  username,
 }: {
   groups: NavGroup[];
   active: ActiveNav | undefined;
+  handleLogout: any;
+  username: string;
 }) => (
-  <Box p={4} minWidth="200px" maxWidth="200px" backgroundColor="#191D27">
-    <VStack as="nav" alignItems="start" color="white" height="100%">
-      <Box pb={6}>
-        <FidesLogoHomeLink />
+  <Box
+    p={4}
+    minWidth="200px"
+    maxWidth="200px"
+    backgroundColor="#191D27"
+    height="100%"
+    overflow="scroll"
+  >
+    <VStack
+      as="nav"
+      alignItems="start"
+      color="white"
+      height="100%"
+      justifyContent="space-between"
+    >
+      <Box>
+        <Box pb={6}>
+          <FidesLogoHomeLink />
+        </Box>
+        <Accordion
+          allowMultiple
+          width="100%"
+          defaultIndex={[...Array(groups.length).keys()]}
+          overflowY="auto"
+        >
+          {groups.map((group) => (
+            <NavGroupMenu key={group.title} group={group} active={active} />
+          ))}
+        </Accordion>
+      </Box>
+      <Box alignItems="center">
```

Comment:
I think this box could use some padding at the bottom for when the menu is fully extended, otherwise the icons end up really close to the bottom of the screen (looks great when menu isn't fully extended tho!) 

but then you might want to change the default `p={4}` on the top level container box to not have a `paddingBottom` or else you'll get double padding in the non-expanded state 

![image](https://github.com/ethyca/fides/assets/24641006/ef48266f-67d4-475a-9142-4704881538de)